scripts/test.sh

```diff
@@ -144,17 +159,25 @@ ENABLE_DATABASE_DELETION=True npx run-s delete-database "fixtures:load-ci cypres
 echo_important "Starting local server with testing env…"
 env $(grep -v '^#' cypress/env/join_with_sponsorship.conf | xargs) DO_NOT_SEND_MAIL=False DO_NOT_RATE_LIMIT=True MONCOMPTEPRO_HOST=$host PORT=$port npm run dev &
```

Comment:
@manuhabitela the only loaded env var is always join_with_sponsorship.conf

Reply:
Woops 😅